docs/overview.rst

```diff
@@ -48,216 +50,298 @@ username and password or using Microsoft Azure *Active Directory* (AD).
 If you don't have a Cloud Console account yet, select the sign-in option you
 wish to use. From there, you will be given the option to sign up.
 
-Once signed in, you will be presented with your :ref:`account overview
-<basics-acc-overview>`.
+Once signed in, you will be presented with your :ref:`organization overview
+<overview-org-overview>` on the Organization page.
 
 
-.. _basics-acc-overview:
+.. _overview-org-overview:
 
-Account overview
-----------------
+Organization
+============
 
-The account overview screen lists the organizations and projects you can
-access.
+The Organization page lists the organizations and projects you can access.
 
 If you follow the `tutorial`_ instructions and deploy a test cluster,
-your account overview will look something like this:
+your organization overview will look something like this:
 
-.. image:: _assets/img/account-overview.png
+.. image:: _assets/img/organization-overviewv2.png
 
-From the main panel, you can view :ref:`organization details <orgs>`, view
-:ref:`project details <project-overview>`, or :ref:`create a new project
-<project-create>`. You can collapse the individual organization widgets, so
-they take up less screen space.
+The Organization screen consists of three tabs: *Settings*, *Users*, and the
+*Audit Log*. By default, it shows the Settings tab, which displays the
+organization name and ID and the email notification settings. You can access
+the other tabs by clicking their respective names at the top.
 
-From the left-hand navigation menu, you can log out. You can also collapse the
-menu.
+By clicking the *Edit* button next to the organization, you can rename it. Here
+you can also set the email address for notifications and indicate whether you
+want to receive them or not. By clicking *Delete*, you can delete the
+organization.
 
+.. warning::
 
-.. _orgs:
+    If you delete your organization, you will delete all projects and clusters
+    associated with it. This action cannot be undone.
```

Comment:
Note. An organization can't be deleted as long as there are still projects in it. A project can't be deleted as long as there are still services in it.